pyglet/lib.py

```diff
@@ -205,11 +208,13 @@ def find_library(self, path):
             search_path.extend([os.path.join(p, libname) for p in self.dyld_library_path])
             search_path.append(path)
             search_path.extend([os.path.join(p, libname) for p in self.dyld_fallback_library_path])
+            search_path.extend([os.path.join(p, libname) for p in self.brew_library_path])
```

Comment:
This seems like it shouldn't be an issue, but I'd double check to make sure this doesn't break conda or venvs.

Reply:
This should be fine as long as it's at the end. Other locations will already have been tried.